pkg/codegen/go/gen_program_test.go

```diff
@@ -129,3 +131,31 @@ func newTestGenerator(t *testing.T, testFile string) *generator {
 	g.Formatter = format.NewFormatter(g)
 	return g
 }
+
+// Tests that "_, err" is first initialized with ":=", and subsequently with "="
+func TestInitErr(t *testing.T) {
```

Comment:
Rather than a test here, should we create a new test .pp program in `testdata` to ensure the output is exactly what we expect?